dev/ci/user-overlays/00496-letouzey-numeral-notation-2.sh

```diff
@@ -0,0 +1,4 @@
+if [ "$CI_PULL_REQUEST" = "496" ] || [ "$CI_BRANCH" = "numeral-notation-2" ]; then
+    HoTT_CI_BRANCH=numeral-overlay-2
+    HoTT_CI_GITURL=https://github.com/letouzey/HoTT.git
```

Comment:
I get a 404, also the PR number is outdated and the branch name is wrong.

Reply:
I think this file should go

Reply:
Gone